build_debian.sh

```diff
@@ -3,26 +3,20 @@
 ## an ONIE installer image.
 ##
 ## USAGE:
-##   ./build_debian USERNAME PASSWORD_ENCRYPTED
-## PARAMETERS:
-##   USERNAME
-##          The name of the default admin user
-##   PASSWORD_ENCRYPTED
-##          The encrypted password, expected by chpasswd command
-
-## Default user
-USERNAME=$1
-[ -n "$USERNAME" ] || {
-    echo "Error: no or empty USERNAME argument"
+##   ./build_debian
+
+## Include configuration files
+. rules/config
+
+# user
+[ -n "$DEFAULT_USERNAME" ] || {
+    echo "Error: no or empty DEFAULT_USERNAME argument"
     exit 1
 }
 
-## Password for the default user, customizable by environment variable
-## By default it is an empty password
-## You may get a crypted password by: perl -e 'print crypt("YourPaSsWoRd", "salt"),"\n"'
```

Comment:
>salt [](start = 76, length = 4)

Could you give an example?
>   1) It uses DES to encrypt accounts password. Furthermore this
>     effectively limits password length to 8 symbols, even if more
>     provided with PASSWORD or DEFAULT_PASSWORD from rules/config

Reply:
Ok, I have updated commit description message to contain more information on Before/After change. It should explain problem.

In short: you can try *any* image and login with only with first 8 symbols from password (e.g. for default image use YourPaSs instead of YourPaSsWoRd).

This works of course only if password for that account isn't changed since image was build (e.g. via passwd(1) or chpasswd(1) tools on running system).